st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
+                "type": "array",
+                "items": {"type": "string"},
+                "description": "Parameter format hidden from public list."
+            },
+            "ack": {
+                "type": "object",
+                "properties": {
+                    "disabled": {"type": "boolean"},
```

Comment:
We use `enabled` attribute everywhere else (resource definitions, rbac definitions, etc.) so if we are going with the object approach we should also use `enabled` here for consistency.